tests/test_v2.py

```diff
@@ -166,6 +164,29 @@ def test_v2_filters_codecs(filters: Any, order: Literal["C", "F"]) -> None:
     np.testing.assert_array_equal(result, array_fixture)
 
 
+@pytest.mark.filterwarnings("ignore")
+@pytest.mark.parametrize("store", ["memory"], indirect=True)
+def test_create_array_defaults(store: Store):
+    """
+    Test that passing compressor=None results in no compressor. Also test that the default value of the compressor
+    parameter does produce a compressor.
+    """
+    g = zarr.open(store, mode="w", zarr_format=2)
+    arr = g.create_array("one", dtype="i8", shape=(1,), chunks=(1,), compressor=None)
+    assert arr._async_array.compressor is None
```

Comment:
```suggestion
    assert arr.compressor is None
```
Does this work, too?

Reply:
Yes, but that's deprecated, apparently

Reply:
So should be `asyncarray.compressor`. You can also check `array.compressors`